aspnetcore/toc.yml

```diff
@@ -53,15 +93,55 @@
       items: 
         - name: EF Core with Razor Pages
           displayName: tutorial
-          uid: data/ef-rp/index
-        - name: EF Core with MVC, existing DB
+          items: 
+            - name: Overview
+              uid: data/ef-rp/index
+            - name: Get started
+              uid: data/ef-rp/intro
+            - name: Create, Read, Update, and Delete
+              uid: data/ef-rp/crud
+            - name: Sort, filter, page, and group
+              uid: data/ef-rp/sort-filter-page
+            - name: Migrations
+              uid: data/ef-rp/migrations
+            - name: Create a complex data model
+              uid: data/ef-rp/complex-data-model
+            - name: Read related data
+              uid: data/ef-rp/read-related-data
+            - name: Update related data
+              uid: data/ef-rp/update-related-data
+            - name: Handle concurrency conflicts
+              uid: data/ef-rp/concurrency
+        - name: EF Core with MVC, existing database
           displayName: tutorial
           href: /ef/core/get-started/aspnetcore/existing-db
-        - name: EF Core with MVC, new DB
+        - name: EF Core with MVC, new database
           displayName: tutorial
           href: /ef/core/get-started/aspnetcore/new-db
-        - name: EF Core with MVC, long tutorial
-          uid: data/ef-mvc/index
+        - name: EF Core with MVC, 10 tutorials
```

Comment:
Shouldn't this read "long tutorial" instead of "10 tutorials"?

Reply:
That's what it was originally, I changed it in both places thinking "10 tutorials" would better indicate what to expect than "long tutorial"